modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -110,6 +151,22 @@ public AspNetCoreServerCodegen() {
                 "Uses the Swashbuckle.AspNetCore NuGet package for documentation.",
                 useSwashbuckle);
 
+        addOption(CLASS_MODIFIER,
+                "Class modifiers such as abstract or partial",
```

Comment:
TODO: Consider making this an enum option.

Reply:
Yup will do thatt Added TODO for that.